apps/storybook/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@umami/storybook",
+  "type": "module",
+  "scripts": {
+    "dev": "storybook dev -p 6006",
+    "build": "storybook build -o ./dist"
+  },
+  "devDependencies": {
+    "@chromatic-com/storybook": "^1.5.0",
+    "@storybook/addon-essentials": "^8.1.10",
+    "@storybook/addon-interactions": "^8.1.10",
+    "@storybook/addon-links": "^8.1.10",
```

Comment:
please put prod dependencies to the 'dependencies', not 'devDependencies' and vice versa

Reply:
this deps are autogenerated by storybook so let's keep it like this